packages/apps/api-authorizer/src/handler.ts

```diff
@@ -52,6 +52,10 @@ export const createLambdaHandler = (validateToken: TokenValidator) => async (
   const authHeader = event.headers?.['Authorization'];
   const token = authHeader?.match(/^Bearer (.*)$/)?.[1];
   console.log(token, '<<<<< TOKEN VALUE');
+  console.log(event, '<<<<< WHOLE API GATEWAY REQUEST AUTHORIZER EVENT');
+  if (token == undefined) {
+    throw new Error('Token is undefined');
```

Comment:
We want a log rather than an error, so we still send the correct HTTP 401 response to the client.